### ISSUE TITLE
GREI 3: Task 1 - Implement filtered search by funder and by research affiliation in each repository using ROR

### ISSUE THREAD

**Issue by cmbz** [IQSS/dataverse-pm#165]
## Overview

- Task 1: Implement filtered search by funder and by research affiliation in each repository using ROR.

## Deliverables
- [ ] Deliverable: Individual repositories ensure the implementation of ROR for affiliations and implementation of ROR for funder information.
- [ ] Deliverable: Individual repositories utilize ROR to power affiliations and funder search facets.

## Depends On
- https://github.com/IQSS/dataverse-pm/issues/127

## Related
- https://github.com/IQSS/dataverse-pm/issues/19
- https://github.com/IQSS/dataverse.harvard.edu/issues/230
- https://github.com/IQSS/dataverse/issues/10196
- https://github.com/IQSS/dataverse/issues/4859
- https://github.com/IQSS/dataverse/issues/9151
- https://github.com/IQSS/dataverse/issues/6640
- https://github.com/IQSS/dataverse/issues/9720
- https://github.com/IQSS/dataverse/issues/9150

## Resources
- https://github.com/IQSS/dataverse-pm/issues/124
- https://github.com/IQSS/dataverse-pm/issues/117
- [APPROVED - NIH GREI Coopetition Plan Year 3](https://docs.google.com/document/d/1XTJSOfUlg06WYGXe3URVFH6VaPMkBNPqVZgrsqOrbc0/edit?usp=sharing)

**Comment by cmbz** [comment]
@jggautier and @sbarbosadataverse is it okay to close this issue now?

**Comment by jggautier** [comment]
Yeah it makes sense to me to close it. The scope seems weird to me. I don't know that all GREI repositories have implemented any sort of filtered search by funder and researcher affiliation using ROR, but more importantly to me, I don't think that folks from any particular GREI repository, like us at Harvard Dataverse, agreed to ensure that each GREI repository do this. Maybe it's just a matter of wording and this issue is just about folks at Harvard Dataverse doing this?

The first deliverable, "Individual repositories ensure the implementation of ROR for affiliations and implementation of ROR for funder information", seems related to the update in v6.6 and the post-update tasks for Harvard Dataverse described at https://github.com/IQSS/dataverse.harvard.edu/issues/326. When that's done we can say that Harvard Dataverse is able to record RORs of funders and the affiliations of "Authors". Of course that doesn't mean that all deposits will have affiliation and funder metadata that includes RORs.

For the second deliverable, "Individual repositories utilize ROR to power affiliations and funder search facets," I've thought that this meant that a user could use the facetted search on the left-hand side of the search page to find deposits published in any collection in Harvard Dataverse that:

- Are authored by anyone from a particular organization, like a university
- Come from funding from a particular organization, like the NIH or one of its centers

And that search would be "powered" by ROR, meaning that the search is using the ROR ID of that particular organization, instead of the funder name like it would be now.

We haven't done what I've thought that this meant, but more importantly to me, we haven't talked about what "utilize ROR to power affiliations and funder search facets" would mean, so I don't know that we have a shared understanding of it and how we'd evaluate the success of whatever solutions we implement.

I think how I've interpreted this is connected to the larger discussion among the GREI group about challenges with repository-wide curation, especially for repositories that have a relatively large number of deposits and use a "self-curation" model. In this case for example, the changes in v6.6 and those post-upgrade tasks won't guarantee that RORs will be included in the metadata, and in the "right" metadata fields, of all deposits, and I'd guess that this will be even less likely for deposits that were created before those changes.

Sorry for this long post! I think I've talked about this stuff in previous meetings but haven't and still don't have the bandwidth to pay as much attention to it.

**Comment by cmbz** [comment]
2025-04-05
- Closing as complete per Julian's comment above: https://github.com/IQSS/dataverse-pm/issues/165#issuecomment-2773087810

**Comment by jggautier** [comment]
I removed this issue from the list at https://github.com/IQSS/dataverse-pm/issues/117#issue-2051574688
